content/docs/concepts/csidriver/features/powerflex.md

```diff
@@ -283,6 +283,18 @@ allowedTopologies:
     - csi-vxflexos.dellemc.com
 ```
 
+**Note** : 
+  The NFS labels are automatically added by all drivers, assuming that NFS dependencies are configured by default. These dependencies come with the default Linux OS package from the node and the array supports NFS. For other protocols (iSCSI, FC, NVMe), all necessary connectivity checks between the initiator and target are performed before labels are added.
```

Comment:
As this is on PowerFlex driver page and as we have added the same under diff diff driver sections. So shall we remove all drivers from here?

Reply:
Updated